hazelcast-client/src/main/java/com/hazelcast/client/proxy/ClientReplicatedMapProxy.java

```diff
@@ -422,7 +422,7 @@ private void addNearCacheInvalidateListener() {
             invalidationListenerId = registerListener(createNearCacheInvalidationListenerCodec(), handler);
         } catch (Exception e) {
             ILogger logger = getContext().getLoggingService().getLogger(ClientReplicatedMapProxy.class);
-            logger.severe("-----------------\n Near Cache is not initialized!!! \n-----------------", e);
+            logger.severe("-----------------\n Near Cache is not initialized! \n-----------------", e);
```

Comment:
No space needed before the second `\n` (and I would also remove it after the first `\n`)